src/client.js

```diff
@@ -347,6 +349,7 @@ export class Saturn {
    */
   async * fetchContent (cidPath, opts = {}) {
     let res, controller, log
+    opts = Object.assign(this.config, opts)
```

Comment:
Don't overwrite `this.config`

```suggestion
    opts = Object.assign({}, this.config, opts)
```